hal/src/nRF52840/lwip/lwip_logging.h

```diff
@@ -28,10 +28,6 @@ void lwip_log_message(const char *fmt, ...);
 }
 #endif /* __cplusplus */
 
-#ifdef DEBUG_BUILD
```

Comment:
This also has a potential to pollute things too much as it enables logging for every LwIP subsystem. What's more this duplicates the TX/RX logs mentioned above. May affect the performance as well (e.g. PPP logs are very verbose in case of integrity issues and will produce a torrent of lines).

Reply:
Hmm, I didn’t see that in testing. What build option would cause this? I tried modular without debug and mono with debug build.

Reply:
Hmm, I didn’t see that in testing. What build option would cause this? I tried modular without debug and mono with debug build.

Reply:
Ah, you are right. My bad. For this to work `LWIP_DEBUG` and other `XXX_DEBUG` options need to be enabled in `lwipopts.h`. So, what does this enable us then..?

Reply:
Maybe nothing :-) originally I was after the “lwip” logs, so I should restore this. Will fix in the morning. Thanks for calling it out.

Reply:
Turns out it wasn't needed, thanks!  Fixed.